### PR TITLE
Add support for specify Cardinality for EventHubTrigger

### PR DESCRIPTION
**Pull request by Flanker32**
Add support for specify Cardinality for EventHubTrigger, for issue #412

### CODE REVIEW

File: azure-functions-maven-plugin/src/main/resources/templates.json [hunk 1]
```diff
@@ -163,13 +163,14 @@
             "name": "myEventHubMessage",
             "direction": "in",
             "eventHubName": "samples-workitems",
+            "cardinality": "many",
```
Comment (jdneo) [inline azure-functions-maven-plugin/src/main/resources/templates.json]: Why we need to add this line here?

File: azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/bindings/EventHubBinding.java [hunk 2]
```diff
@@ -37,6 +43,11 @@ public EventHubBinding(final EventHubOutput eventHubOutput) {
         connection = eventHubOutput.connection();
     }
 
+    @JsonGetter
+    public String getCardinality() {
+        return cardinality.toString().toLowerCase(Locale.getDefault());
```
Comment (jdneo) [inline azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/bindings/EventHubBinding.java]: `Locale.ENGLISH`

**Comment by jdneo** [comment]
LGTM, waiting for `azure-functions-java-library` releasing...

**Comment by jdneo** [comment]
@Flanker32 Please update the dependency in pom also

**Comment by pragnagopa** [comment]
azure-functions-java-library version 1.0.0-beta-7-SNAPSHOT is now published. Please add following in pom.xml
```
<repositories>
		<repository>
			<id>maven.snapshots</id>
			<name>Maven Central Snapshot Repository</name>
			<url>https://oss.sonatype.org/content/repositories/snapshots/</url>
			<releases>
				<enabled>false</enabled>
			</releases>
			<snapshots>
				<enabled>true</enabled>
			</snapshots>
		</repository>		
	</repositories>
```

**Comment by pragnagopa** [comment]
Ping!

**Comment by Flanker32** [comment]
> azure-functions-java-library version 1.0.0-beta-7-SNAPSHOT is now published. Please add following in pom.xml
> 
> ```
> <repositories>
> 		<repository>
> 			<id>maven.snapshots</id>
> 			<name>Maven Central Snapshot Repository</name>
> 			<url>https://oss.sonatype.org/content/repositories/snapshots/</url>
> 			<releases>
> 				<enabled>false</enabled>
> 			</releases>
> 			<snapshots>
> 				<enabled>true</enabled>
> 			</snapshots>
> 		</repository>		
> 	</repositories>
> ```

@pragnagopa  Is script here means maven will only get SNAPSHOT libraries from sonatype? If so, shall we keep this part in pom when we release plugin?

**Comment by pragnagopa** [comment]
Yes correct. When publishing to maven central, reference for azure-functions-java-library will be updated to the version from maven central. I will send a PR after runtime is deployed to prod

**Pull request merged**